<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clients</class>
 <widget class="QDialog" name="Clients">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clientes</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>701</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="Filters">
    <item row="0" column="0">
     <widget class="QLabel" name="srname">
      <property name="text">
       <string>Surname</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="mc">
      <property name="text">
       <string>MAC</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="Surname_filter"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="Filter">
      <property name="text">
       <string>Filtrar</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="Mac_filter"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>330</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Añadir fecha</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="month">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>300</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="year">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>300</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2016</number>
   </property>
   <property name="maximum">
    <number>2084</number>
   </property>
  </widget>
  <widget class="QLabel" name="MonthYear">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>280</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mes/Año</string>
   </property>
  </widget>
  <widget class="QPushButton" name="estados_pagos">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>370</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Modificar Pago</string>
   </property>
  </widget>
  <widget class="QLabel" name="PYT">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>250</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pagos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="alter">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>210</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Modificar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="new_user">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Añadir</string>
   </property>
  </widget>
  <widget class="QLabel" name="CLT">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Clientes:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>50</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="Shows">
    <item row="0" column="0">
     <widget class="QPushButton" name="clientstable">
      <property name="text">
       <string>Clientes</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pays">
      <property name="text">
       <string>Pagos</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>480</y>
     <width>101</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="register_2">
      <property name="text">
       <string>Registrar</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>Cerrar sesion</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="impagos">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>410</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver Impagos</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
